objectserver/ec/indexdb.go

```diff
@@ -400,6 +404,13 @@ func (ot *IndexDB) Lookup(hsh string, shard int, justStable bool) (*IndexDBItem,
 			FROM objects
 			WHERE hash = ? AND shard = ? AND nursery = 0
 		`, hsh, shard)
+	} else if shard == shardAny {
+		rows, err = db.Query(`
+			SELECT timestamp, deletion, metahash, metadata, nursery
+			FROM objects
+			WHERE hash = ? AND metadata IS NOT NULL
+			ORDER BY nursery DESC, shard ASC
+		`, hsh)
```

Comment:
Do you need to select shard as well? Is putting the -1 into the IndexDBItem below useful?